### DISCUSSION TITLE
The mouse does not work on the latest build for Windows.

### DISCUSSION THREAD

**Discussion by Alexandring** [ArthurSonzogni/FTXUI#393] [General]
I am very interested in your library. Downloaded and compiled the source code for Windows. In none of the examples does the mouse work. In the homescreen example, everything jumps and jumps. Could it be related to the graphics card drivers?
I am used VS2019 + Windows 10 x64.
I also noticed the symbols "?" in many examples.

**Comment by ArthurSonzogni** [comment]
Hello! Thanks for testing FTXUI.

I don't have a Windows desktop and I can't really test this. However, I know many people tried this on windows without any issues.

What console are you using? Is this Microsoft one?
Could you try v2.0.0 and check if this previous version was working correctly?

**Reply by Alexandring** [reply]
Yes, this is the Microsoft console. I will try version 2.0. And run tests on another PC.

**Reply by ArthurSonzogni** [reply]
That's weird.

What are you using to compile and link FTXUI? Are you using cmake?

**Comment by Alexandring** [comment]
https://user-images.githubusercontent.com/49656151/166814349-287ca0e6-932f-480d-a110-7bf7ad692bfb.mp4
I tried with version 2.0. The problems are the same. See video...
